interface/weno/rxlogmanager.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Rx\Weno\Container;
+
```

Comment:
need an acl check here (should be the same one that decide to use in the menu link)